helix-rest/src/main/java/org/apache/helix/rest/common/HelixDataAccessorWrapper.java

```diff
@@ -14,7 +14,7 @@
  * The caches is of the value from get methods and short lived for the lifecycle of one rest request
  * TODO: add more cached read method based on needs
  */
-public final class HelixDataAccessorWrapper extends ZKHelixDataAccessor {
+public class HelixDataAccessorWrapper extends ZKHelixDataAccessor {
```

Comment:
I am not sure if HelixDataAccessorWrapper is the ideal name for this class. Perhaps something like `RESTRequestDataAccessor` might be more appropriate and descriptive of what it's actually doing?

Reply:
The class is only visible in rest module thus it implies the usage under helix-rest only. We understand HelixDataAcessor in core module is used to fetch helix related information on zookeeper. I feel wrapper should make sense in this case.